mesh-workshop.bat

```diff
@@ -0,0 +1,26 @@
+@echo off
+
+set confpath=%1
+set nodename=%2
+set HOSTNAMES="bathurst bayview bloor broadview chester christie college coxwell davisville donlands dufferin dundas dupont eglinton ellesmere finch greenwood islington jane keele kennedy king kipling lawrence leslie midland museum osgoode ossington pape queen rosedale runnymede spadina summerhill union warden wilson woodbine yorkdale"
+
+rem Check required inputs
+IF [%confpath%] == [] GOTO VARMISSING
+IF [%confpath%] == [] GOTO VARMISSING
+rem Make sure the node name is valid
+if NOT exist custom\conf.d\%nodename%\30-mesh-workshop.tar.gz goto VARMISSING
+rem Make sure destination is valid
+if NOT exist %confpath%\ goto VARMISSING
```

Comment:
We should validate path _before_ node name, as in other script

Reply:
The reason why I want to validate `conf.d` is to avoid people copying to the wrong directory, not whether they are copying to a real location. So this check isn't really what I intended.